frontend/src/components/Header.tsx

```diff
@@ -7,14 +7,14 @@ const navigate = useNavigate()
 
     function handleLogout() {
         axios.post("/auth/logout").catch(error => console.error(error))
```

Comment:
solltest du den User in deinem Frontend noch in einem State speichern, dann vergiss nicht den auch noch zu leeren.